azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/common/audiodevicelist/AudioDeviceListView.kt

```diff
@@ -108,7 +108,8 @@ internal class AudioDeviceListView(
                     ),
                     null,
                     context.getString(R.string.azure_communication_ui_setup_view_audio_device_selected_accessibility_label),
-                    enabled = initialDevice == AudioDeviceSelectionStatus.RECEIVER_SELECTED
+                    enabled = initialDevice == AudioDeviceSelectionStatus.RECEIVER_SELECTED,
+                    null
```

Comment:
BottomCellItem should use Named params so we can leave these nulls out, and be a bit more clear about what we are passing in (too many params) can follow up in future PR.

Reply:
will do that as part of refactoring, nice suggestion :)